src/EditorFeatures/Core/Shared/SuggestionSupport/DefaultDocumentSupportsSuggestionService.cs

```diff
@@ -22,5 +22,10 @@ public bool SupportsRename(Document document)
         {
             return true;
         }
+
+        public bool SupportsNavigationToAnyPosition(Document document)
```

Comment:
So the name 'SuggestionService' is just wonky now.  This is now hte "DocumentSupportsFeatureService".